lib/request.js

```diff
@@ -50,6 +52,11 @@ function Request (options) {
   this.headers['user-agent'] = this.headers['user-agent'] || 'lightMyRequest'
   this.headers.host = this.headers.host || options.authority || hostHeaderFromURL(parsedURL)
 
+  if (options.cookies) {
+    const { cookies } = options
+    this.headers.cookie = Object.keys(cookies).map(key => cookie.serialize(key, cookies[key])).join('; ')
```

Comment:
I think we should document the fact that the cookies option take precedence over the cookie header.

Reply:
Maybe we could just append, so the user will decide